packages/web/tests/splunk-otel-web.test.ts

```diff
@@ -86,11 +96,14 @@ describe('SplunkOtelWeb', () => {
 				key3: 'value3',
 			})
 
-			expect(receivedAttributes).toStrictEqual({
+			expect(receivedAttributes).toMatchObject({
 				key1: 'value1',
 				key2: 'value2-changed',
 				key3: 'value3',
 			})
+			// Should also include platform attributes
+			expect(receivedAttributes).toHaveProperty('user_agent.os.name')
+			expect(receivedAttributes).toHaveProperty('user_agent.language')
```

Comment:
Everywhere else, you also check for `user_agent.original`. Is there any reason why you don’t do it here? Just wondering.